v2/src/components/api/compatibility.ts

```diff
@@ -11,18 +11,13 @@ export type GetCompatibilityResponse = {
     driverToFrontend: { [key: string]: string[] };
 };
 
```

Comment:
there is an issue about shadow drop or something when something went wrong is shown

Reply:
done